selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/opendeclaration/Eclipse0093Test.java

```diff
@@ -48,6 +50,7 @@
   @Inject private CodenvyEditor editor;
   @Inject private TestProjectServiceClient testProjectServiceClient;
   @Inject private BrowserLogsUtil browserLogsUtil;
+  @Inject private SeleniumWebDriver webDriver;
```

Comment:
it's better to avoid injection of WebDriver into the test. Lets inject webdriver in the same way as it was done in **SeleniumWebDriverHelper** class.